### DISCUSSION TITLE
FPM and VIA´s

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5870] [General]
Hi dev team,

as far as I understand is the FPM in the dev mode integrated? If yes There is a huge issue (I guess you heard about this :-P) with the STAR and espacially with the VIA´s. When selecting a Star and programming the star with the option "no VIA", the VIA is still in the box .. In this case it is the deleted VIA BADSO in EDDS ... 

![grafik](https://user-images.githubusercontent.com/76531893/134147892-9489099f-4fc0-4255-bc1d-4135b05216a0.png)

![grafik](https://user-images.githubusercontent.com/76531893/134147957-c330395a-7b79-4c5b-ba6e-3ed5098c3fce.png)

Greets

**Comment by tracernz** [comment]
No, the new FPM and guidance system is only in the experimental version. I recommend you give it a try. :)

**Reply by CptBayer** [reply]
Thanx .... rock´n roll ... I will try ...

**Reply by CptBayer** [reply]
FAN-TAS-TIC !!!!!!! I love it ..... thanx a lot for that improvement !!!!

**Reply by tracernz** [reply]
It's still got plenty of work to be done, but already a big improvement in most cases. Hope your flights are more enjoyable now. :D
